### PR TITLE
[FIRE EMBLEM by Dylan B] Bugfix Menu Bar changed and image links changed to point github repository

### PR DESCRIPTION
**Pull request by VJRolmasters**
# Submission Checklist
## Required

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

# Changes / Description

- Bugfix Menu Bar changed : In order to solve the problem with the menu bar in firefox navigator I have changed all the menu bar for a new one (html + css)

- I have redo all the links to images so they point to github repository.

### CODE REVIEW

File: Fire Emblem/Fire-emblem.html [hunk 1]
```diff
@@ -477,84 +458,67 @@
...
                                   </div><!-- end .sheet-width100 --> 
                             
                                   <div class="sheet-width100"> 
                                              <div class="sheet-combatA">CRITICAL</div><!-- end .sheet-combatA --> 
 
-                                             <div class="sheet-combatA"> 
-                                                           <input type="number" name="attr_total_critical" style="width:100px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="0" />     
-                                             </div><!-- end .sheet-combatA --> 
+                                             <div class="sheet-combatB"> 
+                                                           <input type="number" name="attr_total_critial" style="width:100px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="0" />
```
Comment (NorWhal) [inline Fire Emblem/Fire-emblem.html:488]: looks like you have a typo here

File: Fire Emblem/Fire-emblem.html [hunk 2]
```diff
@@ -800,89 +760,71 @@
...
-                                   <input type="number" name="attr_sleight_total" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="@{sleight_statsnumber}+@{sleight_points}" disabled="true" />
+                                   <input type="number" name="attr_sleight_total" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;"/>
                                    <input type="text" name="attr_sleight_stats" style="width:225px; height: 25px; margin: 0px 5px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="SKL + LUK/2" readonly/> 
-                                  <input type="hidden" name="attr_sleight_statsnumber" style="width:95px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="[[@{skl}+@{luk}/2]]" disabled="true" />
-                                   <input type="number" name="attr_sleight_points" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="0" />
```
Comment (NorWhal) [inline Fire Emblem/Fire-emblem.html:873]: A lot of the `_statsnumber` and `_points` attrs are being removed, and it appears they're being replaced with `_level`? I'm not sure what their function is, but we can't approve an update that would delete players' data without handlers to convert it into their replacements.

If these are somehow just UI attributes, I'll be happy to approve this after an explanation, but the names lead me to believe they have some mechanical importance.

**Review by NorWhal — COMMENTED** [review]
Hey @VJRolmasters!

I have one concern that I'd like to hear back about before I can approve this one. I'll keep an eye on this and try to get back ASAP!

Thanks,
Nora

**Comment by VJRolmasters** [comment]
Hi Norwhal 

Thanks, I really don't know what I did. The files seems a mess there. So I did it again and I uploaded the files again. New code of HTML and CSS are just to change the menu bar (the old one wasn't workink in Firefox) and the links of the images to point the Github folder.

Thanks again

**Comment by NorWhal** [comment]
@VJRolmasters Understood!

This most recent commit (519b2d4) added files at the root level of the repository, which can't we approve. Once that is amended, we can go ahead and merge this. I'll try to keep an eye on this PR and get it approved once that change is in.

**Comment by VJRolmasters** [comment]
Ok I am off.

I close this pull request and start a new one. I didn't know how to solve this.

Sorry for all troubles

**Pull request closed without merging**